lib/components/user/nav-login-button.js

```diff
@@ -78,21 +79,28 @@ export default class NavLoginButton extends Component {
 
           {links && links.map((link, i) => (
             <LinkContainerWithQuery exact key={i} target={link.target} to={link.url}>
-              <MenuItem>{link.text}</MenuItem>
+              <MenuItem>
+                {link.messageId === 'myAccount' // messageId is 'myAccount' or 'help'
```

Comment:
Especially given your comment here I would use a template string to generate the id, but given how prevalent this pattern is it makes me think that perhaps generating ids with template strings is considered bad practice? If so please ignore all my comments.